ablator/main/model/wrapper.py

```diff
@@ -612,7 +612,8 @@ def train_loop(self, smoke_test=False):
                 self.metrics.reset("train")
                 self.train_tqdm.reset()
             outputs, train_metrics = self.train_step(batch)
-            self.metrics.append_batch(**outputs, tag="train")
+            if outputs is not None:
```

Comment:
is there a test for this change?

Reply:
Yes it's fixed [here](https://github.com/fostiropoulos/ablator/blob/07e43eed45bb12d683471856c25b907af9da10d6/tests/main/test_wrapper.py#L394)